R/vars_manager.R

```diff
@@ -49,10 +49,20 @@ get_single_vector_var <- function(obj, name) {
     ))
 }
 
-get_vector_var <- function(obj, name, abbrev_len=DEFAULT_ABBREV_LEN) {
+get_vector_var <- function(obj, name, abbrev_len=DEFAULT_ABBREV_LEN, sort_by=NULL, ascending=NULL) {
+    # sort_by should be a single string, "value"
+    # ascending should be a single boolean
     summary <- sprintf("Length: %d", length(obj))
     abbrev <- !is.null(abbrev_len) && length(obj) > abbrev_len
-    obj_pre <- `if`(abbrev, obj[1:abbrev_len], obj)
+
+    if (is.character(sort_by) && tolower(sort_by) == "value") {
+        decreasing <- !`if`(is.logical(ascending), ascending, TRUE)
+        obj_sorted <- sort(obj, decreasing=decreasing)
```

Comment:
looks like `sort` defaults to removing NA values, which seems weird? (https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/sort). I think we'll want a `na.last = decreasing` here

Reply:
That's a great catch, I thought the default was just to stick NAs at the end, not get rid of them. I added a test for that too